### PR TITLE
 wiki: support wiki-casting on non-inventory items 

### LINKED ISSUE #7854: Cannot select items in bank for wiki lookup
**Describe the bug**
Items in the bank cannot be targeted for wiki lookup.

**To Reproduce**
1. Enable the wiki plugin.
2. Open the bank.
3. Click on the `Lookup Wiki` option on the wiki button under the minimap.
4. Click on any item in the bank.
5. Observe that nothing happened.

**Expected behavior**
The item in the bank would be selected and then a web page for the wiki would open up showing the article for the selected item. Basically, it would work exactly like the lookup feature works on items in the inventory.

**Environment (please complete the following information):**
 - OS: Windows
 - RuneLite version: 1.5.12
 - Launcher version: 1.6.0

### PR DESCRIPTION
**Pull request by abextm**
Closes #7854

Depends on !79

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java [hunk 1]
```diff
@@ -245,6 +248,18 @@ private void onMenuOptionClicked(MenuOptionClicked ev)
...
+
+					if (w.getType() == WidgetType.GRAPHIC && w.getItemId() != -1)
+					{
+						type = "item";
+						id = itemManager.canonicalize(w.getItemId());
+						name = itemManager.getItemComposition(id).getName();
+						location = null;
+						break;
+					}
+					// fallthrough
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java]: Is this correct? There are definitely non-item graphics.

Comment (abextm) [reply runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java]: Thats why I check `itemID != -1`

Comment (Adam-) [reply runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java]: Right, but if not it will fallthrough and log a warning about it

Comment (abextm) [reply runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java]: It shouldn't have gotten a menuop if its not an item, so logging a warning is appropriate.

**Pull request merged**